app/src/main/java/com/joseg/fakeyouclient/common/enums/Enums.kt

```diff
@@ -0,0 +1,21 @@
+package com.joseg.fakeyouclient.common.enums
+
+enum class TtsRequestStatusType {
```

Comment:
Added an `Enum` class to represent the statuses value that a `TtsRequest` object can undergo.